examples/index.html

```diff
@@ -27,6 +27,7 @@
   <script src="../src/ui/prepareDynamic.js"></script>
   <script src="../dist/image-sequencer.js" charset="utf-8"></script>
   <script src="../dist/image-sequencer-ui.js" charset="utf-8"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-colorpicker/2.5.3/js/bootstrap-colorpicker.js"></script>
```

Comment:
When it is ready. Please change src pointing to package in node modules. As I can see you have already included the package.

Good choice of library. 😍

Reply:
Actually both these things are needed for incorporating the plugin.

Reply:
Can it not be included through node modules? Isn't the library available in node? @aashna27?

Reply:
Yeah I have made those changes! Thanks a lot :+1: @jainaman224 @HarshKhandeparkar